### PR TITLE
feat delay friendship accept

### PR DESCRIPTION
**Pull request by SilentQianyi**
## I'm submitting a...

```
[ ] Bug Fix
[ ] Feature
[ ] Other (Refactoring, Added tests, Documentation, ...)
```

## Checklist

- [ ] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
- [ ] Tests for the changes have been added

## Description

_please describe the changes that you are making_

_for features, please describe how to use the new feature_

_please include a reference to an existing issue, if applicable_

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```
Issue
https://github.com/wechaty/wechaty/issues/1890

### CODE REVIEW

File: src/puppet-padplus.ts [hunk 1]
```diff
@@ -374,7 +374,7 @@ export class PuppetPadplus extends Puppet {
     }
     const payload = await this.manager.getFriendship(friendshipId)
     if (!payload || payload.type !== FriendshipType.Receive) {
-      throw new Error(`friendship type error.`)
+      throw new Error(`can not find friendship payload ${payload} or friendship type ${payload?.type} error.`)
```
Comment (su-chang) [inline src/puppet-padplus.ts]: ${payload} you can only get a string like `[Object object]`

File: src/puppet-padplus.ts [hunk 2]
```diff
@@ -400,6 +400,15 @@ export class PuppetPadplus extends Puppet {
     return rawPayload as FriendshipPayload
   }
 
+  protected async friendshipPayloadSave (FriendshipPayload: FriendshipPayload): Promise<FriendshipPayload | void> {
```
Comment (su-chang) [inline src/puppet-padplus.ts]: Let's make the variable `FriendshipPayload` start with lowercase.

File: src/puppet-padplus.ts (hunk 1 above)
Comment (windmemory) [reply src/puppet-padplus.ts]: agree

Comment (SilentQianyi) [reply src/puppet-padplus.ts]: fix

**Pull request merged**